core/src/repair/serve_repair.rs

```diff
@@ -1249,14 +1249,13 @@ impl ServeRepair {
             }
         }
         if !pending_pongs.is_empty() {
-            match batch_send(repair_socket, &pending_pongs) {
+            let num_pkts = pending_pongs.len();
+            let pending_pongs = pending_pongs.iter().map(|(bytes, addr)| (bytes, addr));
```

Comment:
this looks redundant... why map without any actual transform?

Reply:
The closure is converting `&(S, T)` to `(&S, &T)`.
The code will not build without it.

Reply:
Oh I see. rust sometimes works in mysterious ways!